CHANGES.md

```diff
@@ -6,6 +6,7 @@ New Languages:
 
 Language improvements:
 
+- enh(cpp): Support C++ pack expansion in function arguments
```

Comment:
Please add your name at the end here and link around line ~31 or so.

Reply:
Done.